.github/workflows/javascript-tests.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: JavaScript tests (Jest)
+# This CI workflow uses Jest, which runs in a Node.js environment, to test the JavaScript code.
+# Jest allows JavaScript modules to be loaded and tested in isolation.
+# By contrast, the Playwright CI workflow uses Playwright, which loads the entire app in the browser and simulates user interactions.
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
```

Comment:
Should this be `20.x`? Package.json for this repo says:

```
  "engines": {
    "node": ">=18.0.0"
  }
```

Reply:
yes we should be testing against `18.x` and `20.x`